spyderlib/widgets/ipython.py

```diff
@@ -628,6 +632,23 @@ def clear_line(self):
     def clear_console(self):
         """Clear the whole console"""
         self.shellwidget.execute("%clear")
+        
+    @Slot()
+    def reset_namespace(self):
+        """Resets the namespace by removing all names defined by the user"""
+        
+        reply = QMessageBox.question(
+            None,
+            _("Reset IPython namespace"),
+            _("All user-defined variables will be removed."
+            "<br>Are you sure you want to clean the namespace?"),

```

Comment:
I mean, this line should read

    Are you sure you want to reset the namespace?